<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_start">
    <property name="minimumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <widget class="QPushButton" name="game_running">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>610</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>开始游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_quit">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>610</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>退出游戏</string>
     </property>
    </widget>
    <widget class="QLabel" name="start_photo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1366</width>
       <height>768</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>start_photo</zorder>
    <zorder>game_running</zorder>
    <zorder>game_quit</zorder>
   </widget>
   <widget class="QWidget" name="page_movePhoto">
    <widget class="QLabel" name="show_move">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1366</width>
       <height>768</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="dop_btn">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>650</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>跳过</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_choise">
    <property name="minimumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <widget class="QPushButton" name="ka_num1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第一关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1photo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_return_start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2photo">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第二关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num3">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第三关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第五关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3photo">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>150</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5photo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>450</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6photo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>450</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>390</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第六关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7photo">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>450</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num7">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>390</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第七关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num4">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第四关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4photo">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>150</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8photo">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>450</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ka_num8">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>390</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>第八关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_choisePhoto">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1366</width>
       <height>768</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_choisePhoto</zorder>
    <zorder>pushButton_return_start</zorder>
    <zorder>label_2photo</zorder>
    <zorder>ka_num3</zorder>
    <zorder>ka_num5</zorder>
    <zorder>label_3photo</zorder>
    <zorder>label_5photo</zorder>
    <zorder>label_6photo</zorder>
    <zorder>ka_num6</zorder>
    <zorder>label_7photo</zorder>
    <zorder>ka_num7</zorder>
    <zorder>label_1photo</zorder>
    <zorder>ka_num4</zorder>
    <zorder>label_4photo</zorder>
    <zorder>label_8photo</zorder>
    <zorder>ka_num8</zorder>
    <zorder>ka_num1</zorder>
    <zorder>ka_num2</zorder>
   </widget>
   <widget class="QWidget" name="page_num1">
    <property name="minimumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <widget class="QLabel" name="label_text_time_2">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>游戏时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>150</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>第一关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_help">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>390</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>帮助</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_num_text_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>点击次数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_start">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>480</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gam_exit">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>480</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_num">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>300</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_return_choise">
     <property name="geometry">
      <rect>
       <x>1234</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdTime">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>300</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Hex</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1366</width>
       <height>768</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelPlay</zorder>
    <zorder>label_text_time_2</zorder>
    <zorder>label_name</zorder>
    <zorder>game_help</zorder>
    <zorder>label_num_text_2</zorder>
    <zorder>game_start</zorder>
    <zorder>gam_exit</zorder>
    <zorder>label_num</zorder>
    <zorder>pushButton_return_choise</zorder>
    <zorder>lcdTime</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
